lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -198,6 +198,42 @@ def _loop_process_output(self, output):
             self.log.error(warning_msg + '\n')
             self.seen_output = True
 
+class RunAsyncMPI(RunAsyncLoopStdout):
+    """
+    RunAsync for MPI in case a program 'hangs'
```

Comment:
this doesn't really have anything to do with hanging, it's just a matter of producing useful output as soon as it's available, mainly for interactive use...